### DISCUSSION TITLE
/items/*** Return more data

### DISCUSSION THREAD

**Discussion by zhuyifang** [directus/directus#7855] [Archive]
This is one of my data-models named: user_focus.
Among them, user_created and user_id are respectively associated with directus_user.id
The setting of interface viewing permissions is as follows:
`{
     "_or": [
         {
             "user_created": {
                 "_eq": "$CURRENT_USER"
             }
         },
         {
             "user_id": {
                 "_eq": "$CURRENT_USER"
             }
         }
     ]
}`

![image](https://user-images.githubusercontent.com/16346219/132213838-6f465958-6112-4c5d-a5bf-849d6ff09613.png)

When I get data through the interface /items/user_focus, the result obtained is:
![image](https://user-images.githubusercontent.com/16346219/132214295-c4a621f1-8eed-4fc6-b4fb-7c30c7da3211.png)

Is it possible to return more information in directus_user, such as avatar?

**Comment by zhuyifang** [comment]
What needs to be done to return this format? 
```
{
"data":[
  {
    "date_created":"2021-09-06T08:13:18.000Z",
    "id":1,
    "user_created":"9bd77b9c-16df-4c67-86d4-255862b57bcf",
    "user_id":"daa79475-9568-4f9b-bbc7-eca5421152d5"，
    "directus_user":{****}
  }
]}
```

**Comment by zhuyifang** [comment]
@rijkvanzanten Can you help me?

**Comment by biapar** [comment]
I see that if there is a relation and use ***, the 1st level return related data and the 2nd the same

**Reply by zhuyifang** [reply]
First of all thank you very much for your help, but sorry, I did not understand
Can you give an example? Because I tried: `/items/user_focus/?fields=**`
The result is [], in fact the database has data

**Reply by rijkvanzanten** [reply]
@zhuyifang See https://docs.directus.io/reference/api/query/#fields. The syntax is `?fields=*.*`, not `**` 👍🏻
